### PR TITLE
added animation on new high score and hide multiplier on game over

### PR DESCRIPTION
**Pull request by LiamCarter**

### CODE REVIEW

**Review by arxra — CHANGES_REQUESTED** [review]
variable name "highscore" for keeping track of if the score is higher than ighscore animation has play?

File: Assets/Scripts/Score.cs [hunk 1]
```diff
@@ -6,6 +6,8 @@ public class Score : MonoBehaviour {
 
   [Tooltip("The current players score")]
     public int score;
+    public Animator anim;
+    private bool highscore = false;
```
Comment (arxra) [inline Assets/Scripts/Score.cs]: Replace "highscore" with "_higherScore" ?

**Review by arxra — CHANGES_REQUESTED** [review]

**Comment by arxra** [comment]
@perfah @PersikaSemlan @fredrikhanses @JoarEkelund @johansettlin

File: Assets/Scripts/Score.cs [hunk 2]
```diff
@@ -7,10 +7,10 @@ public class Score : MonoBehaviour {
   [Tooltip("The current players score")]
     public int score;
     public Animator anim;
-    private bool highscore = false; 
+    private bool TheBoolThatTelsIfTheAnimationForANewHighScoreHasBeenPlayed = false;
```
Comment (arxra) [inline Assets/Scripts/Score.cs]: @perfah

Comment (perfah) [reply Assets/Scripts/Score.cs]: Spelling mistake / 10

**Review by arxra — APPROVED** [review]

**Pull request merged**